prow/cluster/jobs/all-periodics.yaml

```diff
@@ -63,9 +63,9 @@ periodics:
   name: daily-performance-benchmark-release-1.5
   branches: release-1.5
   decorate: true
-  timeout: 17h
+  timeout: 24h
   decoration_config:
-    timeout: 17h0m0s
+    timeout: 24h0m0s
```

Comment:
Yeah, as we add more tests, we need to extend the timeouts. Well, since we are now only running two perf jobs, i.e. release-1.5 and master and we have three clusters https://github.com/istio/test-infra/blob/master/boskos/resources.yaml#L48. We should  still fine at current stage. But, yeah, nice to do some optimizations later on.